devito/passes/clusters/blocking.py

```diff
@@ -296,8 +296,7 @@ def decompose(ispace, d, block_dims):
 
         for bd in block_dims:
             # Avoid e.g. `x > yb`
-            if any(i._depth > bd._depth for i in r[:n] if i.is_Block) or \
-               any(bd._depth < i._depth for i in r[n+1:] if i.is_Block):
+            if any(i._depth > bd._depth for i in [*r[:n], *r[n+1:]] if i.is_Block):
```

Comment:
Can you elaborate?

Reply:
It is equivalent to before, nothing really changes.

Reply:
I'm struggling with the current syntax honestly?
I find the other more verbose but much easier to understand (@mloubout wdyt?)

Reply:
Oops, I'm dumb... I can see the relationals in the two lines are exactly the same just sign and lhs and rhs swapped (a>b == b<a), lol